templates/content/field-group-html-element--node--oe-publication--group-description.html.twig

```diff
@@ -0,0 +1,10 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a fieldgroup html element.
+ *
+ * @see field-group-html-element.html.twig
+ */
+#}
+
+  {{children}}
```

Comment:
There is no styling applied, {%- include 'field--bare.html.twig' -%} can't be used?